health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -208,17 +263,17 @@
-    
+
             // Process options
             if (question.options) {
                 question.options.forEach((option, optionIndex) => {
                     const optionval = option.label;
                     const upperCaseString = optionval.toUpperCase();
                     const transformedString = upperCaseString.replace(/ /g, '_');
-                    
+
                     if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
                     let formattedStringTemp = `${campaignName}.${checklistTypeTemp}.${roleTemp}.${transformedString}`;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'formattedStringTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)